src/web-pathing.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Web Pathing Specification
+description: >
+  Specification defines a subset of possible content paths that ensures
+  compatibility with existing HTTP and Web Platform standards.
+date: 2023-11-12
+maturity: wip
+editors:
+  - name: Marcin Rataj
+    github: lidel
+    url: https://lidel.org/
+    affiliation:
+        name: Protocol Labs
+        url: https://protocol.ai/
+tags: ['architecture', 'httpGateways', 'webHttpGateways']
+---
+
+Web Pathing Specification defines a subset of possible content paths
+that ensures compatibility with existing HTTP and Web Platform standards.
+
+## Introduction
+
+TODO: Clearly explain why the specification exists, what is the problem solved here.
+
+This document specifies details of pathing for content paths that start with
+`/ipfs` and `/ipns` namespaces, and why a logical content root included in a
+content path can facilitate security isolation and relative pathing in web
+contexts.
+
+Specification includes guidance around aspects such as hash functions,
+multibases, CID versions, codecs, and how they impact implementation's ability
+to translate pathing into traversal of a DAG.
+
+The goal of this specification is to enable competing and interoperable
+implementations, all while ensuring seamless traversal of paths within the web
+ecosystem.
+
+## Specification
+
+TODO: Explain things in depth.
+The resulting specification should be detailed enough to allow competing,
+interoperable implementations.
+
+### TODO: things to cover
```

Comment:
cc @Stebalien @dignifiedquire @hacdias @aschmahmann @Jorropo @rvagg @ribasushi @alanshaw @2color @autonome  @darobin for visibility and sourcing early feedback on the scope of this spec.

Feel free to drop a comment about any tricky/painful pathing edge cases you've encountered over the years that we should clarify web behavior for by including them in this spec :pray:

Reply:
I think it's important to clarify how this differs, overlaps, or varies from pathing defined in https://specs.ipfs.tech/http-gateways/path-gateway, https://specs.ipfs.tech/http-gateways/trustless-gateway, & https://specs.ipfs.tech/http-gateways/subdomain-gateway.